shell/browser/api/electron_api_app.cc

```diff
@@ -1442,6 +1442,28 @@ void App::SetUserAgentFallback(const std::string& user_agent) {
   ElectronBrowserClient::Get()->SetUserAgent(user_agent);
 }
 
+#if defined(OS_WIN)
+
+typedef BOOL(WINAPI* LPFN_ISWOW64PROCESS2)(HANDLE, PUSHORT, PUSHORT);
```

Comment:
use this instead
```
  auto IsWow64Process2 =
      reinterpret_cast<decltype(&::IsWow64Process2)>(
          GetProcAddress(GetModuleHandle(L"kernel32.dll"),
                         "IsWow64Process2"));
```

Reply:
similar existing code:
https://github.com/electron/electron/blob/c68c65f383f7b210c2ad93715dd70dbb9dcb4798/shell/browser/api/electron_api_power_monitor_win.cc#L47-L50